AntennaTracker/wscript

```diff
@@ -8,6 +8,7 @@ def build(bld):
         vehicle=vehicle,
         libraries=bld.ap_common_vehicle_libraries() + [
             'PID',
+            'AP_Avoidance',
```

Comment:
I don't understand the purpose of this here, or why PID is there either. What is special about these libraries?